### PR TITLE
15.0 mig limit to employees

### PR DESCRIPTION
**Pull request by nikunjantala**

### CODE REVIEW

**Comment by jans23** [comment]
@dsolanki-initos Please review.

**Comment by jans23** [comment]
Please create a PR to OCA.

**Comment by jans23** [comment]
any update?

File: limit_to_employees/__init__.py [hunk 1]
```diff
@@ -0,0 +1,2 @@
+# -*- coding: utf-8 -*-
```
Comment (manengel) [inline limit_to_employees/__init__.py]: This should not be necessary.

File: limit_to_employees/__manifest__.py [hunk 2]
```diff
@@ -0,0 +1,31 @@
+# -*- coding: utf-8 -*-
+{
+    'name': "Limit To Employees",
+
+    'summary': """
+        Limit To Employees""",
+
+    'description': """
+       When user do following action in odoo, it only shows internal users contacts in dropdown.
```
Comment (manengel) [inline limit_to_employees/__manifest__.py]: "When users do the following actions..."

File: limit_to_employees/models/__init__.py [hunk 3]
```diff
@@ -0,0 +1,3 @@
+# -*- coding: utf-8 -*-
```
Comment (manengel) [inline limit_to_employees/models/__init__.py]: Please remove.

File: limit_to_employees/models/res_partner.py [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+
+
+class ResPartner(models.Model):
+    _inherit = "res.partner"
+
+    @api.model
+    def get_mention_suggestions(self, search, limit=8):
+        """ override the default method and customize the search filter"""
+        search_dom = expression.OR([[('name', 'ilike', search)], [('email', 'ilike', search)]])
+        search_dom = expression.AND([[('active', '=', True)], search_dom])
```
Comment (manengel) [inline limit_to_employees/models/res_partner.py]: From what I see here, you look up if there is a user like the search input and if they are active.
So, in this context: Employees are activated partners with a user attached? Is this the logic you like to express?

**Review by manengel — CHANGES_REQUESTED** [review]
Left a review feedback.

File: limit_to_employees/__manifest__.py (hunk 2 above)
Comment (nileshsheliya) [reply limit_to_employees/__manifest__.py]: There is description written in the next line point by point.

File: limit_to_employees/models/res_partner.py (hunk 4 above)
Comment (nileshsheliya) [reply limit_to_employees/models/res_partner.py]: It shows the Internal users in dropdown those are active. Yes, in Employee form its related user is set, so in dropdown it shows only  Employees. But if Internal user employee is not created, then also that user is showing in that dropdown.

File: limit_to_employees/models/__init__.py (hunk 3 above)
Comment (nileshsheliya) [reply limit_to_employees/models/__init__.py]: Changes done.

File: limit_to_employees/__init__.py (hunk 1 above)
Comment (nileshsheliya) [reply limit_to_employees/__init__.py]: Yes, Its removed.

File: limit_to_employees/__manifest__.py (hunk 2 above)
Comment (manengel) [reply limit_to_employees/__manifest__.py]: Yes, I saw that. Just gave a hint how to fix the grammar there.

File: limit_to_employees/README.rst [hunk 5]
```diff
@@ -0,0 +1,13 @@
+===========================
+Limit To Employees
+===========================
+
+When user do following action in odoo, it only shows internal users contacts in dropdown.
+  * Chatter mentioning (@username)
+  * Helpdesk ticket assignments
+  * Invoicing, Salesperson
```
Comment (manengel) [inline limit_to_employees/README.rst]: Weren't there originally listed more? What happened to them?

File: limit_to_employees/__manifest__.py (hunk 2 above)
Comment (nileshsheliya) [reply limit_to_employees/__manifest__.py]: Ok, Sorry, I got it.

**Comment by manengel** [comment]
@nikunjantala Any news related to that?

File: limit_to_employees/README.rst (hunk 5 above)
Comment (nileshsheliya) [reply limit_to_employees/README.rst]: I not understand what changes we need to do in description?

Comment (nikunjantala) [reply limit_to_employees/README.rst]: I not understand what changes we need to do in description?

Comment (manengel) [reply limit_to_employees/README.rst]: I think it must have been a mistake on my end.

**Review by manengel — APPROVED** [review]
Looks good to me now.

**Pull request merged**